test/e2e/scenarios/node-labels.go

```diff
@@ -79,6 +79,9 @@ func labeledDeployTestSuite() {
 			}
 		}
 
+		// wait till operator reconcile csi
```

Comment:
Could you please post some comments why we need to wait 15 seconds here?

Reply:
@mishoyama updated